test/test_quantized.py

```diff
@@ -886,19 +886,21 @@ def test_cat_nhwc(self, X, relu):
         torch.testing.assert_allclose(out.dequantize(), ref.dequantize())
         self.assertNotEqual(out.stride(), sorted(out.stride()))
 
-    @given(X=hu.tensor(shapes=hu.array_shapes(min_dims=3, max_dims=3,
-                                              min_side=1, max_side=2),
-                       qparams=hu.qparams()),
-           dim=st.integers(1, 2))
-    def test_mean(self, X, dim):
-        X, (scale, zero_point, torch_type) = X
-        qX = torch.quantize_per_tensor(torch.tensor(X).float(), scale, zero_point, torch_type)
-
-        Y = torch.mean(qX.dequantize(), dim)
-        Y = torch.quantize_per_tensor(Y, scale, zero_point, torch_type).dequantize()
-        qY = torch.mean(qX, dim)
-
-        self.assertEqual(Y, qY.dequantize())
+    @given(X=hu.tensor(shapes=hu.array_shapes(min_dims=4, max_dims=4,
```

Comment:
why do we change instead of adding new test cases? was it only for qnnpack before?

Reply:
We want to eventually unify tests for qnnpack and server side. For cases like mean is it pretty straightforward to merge them.